src/components/dapp-staking-v2/my-staking/items/UnbondingItem.vue

```diff
@@ -9,31 +9,97 @@
       </thead>
       <tbody>
         <tr>
-          <td>{{ item.unbondingAmount.toLocaleString() }} ASTR</td>
+          <td><format-balance :balance="item.amount.toString()" /></td>
           <td>
             <div class="row--remaining-era">
-              <div class="val-era">{{ item.remainingEra }}</div>
-              <astar-irregular-button :width="77" :height="20">Re-bond</astar-irregular-button>
+              <div class="val-era">{{ item.unlockEra.toHuman() }}</div>
+              <astar-irregular-button
+                :width="77"
+                :height="20"
+                @click="showRebondDialog(item.amount.toString())"
+                >Re-bond</astar-irregular-button
+              >
             </div>
           </td>
         </tr>
       </tbody>
     </table>
     <div class="row--manage">
-      <astar-button :width="97" :height="24" :disabled="!item.isEnabled">Withdraw</astar-button>
+      <astar-button
+        :width="97"
+        :height="24"
+        :disabled="item.erasBeforeUnlock !== 0"
+        @click="showWithdrawDialog(item.amount.toString())"
+        >Withdraw</astar-button
+      >
     </div>
+
+    <Teleport to="#app--main">
+      <div :class="'highest-z-index'">
+        <ModalWithdraw
```

Comment:
We should use kebab-case.
https://github.com/AstarNetwork/astar-apps/pull/533#discussion_r991776056